stdlib/Markdown/src/Common/inline.jl

```diff
@@ -149,10 +149,9 @@ end
 const _email_regex = r"^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$"
 
 function _is_mailto(s::AbstractString)
-    length(s) < 6 && return false
-    # slicing strings is a bit risky, but this equality check is safe
-    lowercase(s[1:6]) == "mailto:" || return false
-    return occursin(_email_regex, s[6:end])
+    length(s) < 7 && return false
+    lowercase(first(s, 7)) == "mailto:" || return false
```

Comment:
```suggestion
    startswith(s, "mailto:") || return false
```
Or I guess we could just include the `mailto:` in the regex (and rename it to, say, `_mailto_regex`). I don't think it's used anywhere else.

Reply:
:+1: I guess I was trying to have the change that was as close as possible to the current situation but this makes more sense, will do this